packages/forms/resources/views/components/color-picker.blade.php

```diff
@@ -67,7 +67,7 @@
                             'autocomplete' => 'off',
                             'disabled' => $isDisabled,
                             'id' => $getId(),
-                            'placeholder' => $getPlaceholder(),
+                            'placeholder' => ($getPlaceholder() === null) ? null : e((string) $getPlaceholder()),
```

Comment:
Please ensure that `$getPlaceholder()` is not called more than once per view, you can create a `$placeholder` variable like in `tags-input.blade.php`

Please refactor each condition to `filled($placeholder) ? e($placeholder) : null`